shared/tasks/build/package.test.mjs

```diff
@@ -70,6 +70,12 @@ describe('package/', () => {
         return output
       }))
 
+      // Add Autoprefixer prefixes to all source '*.scss' files
+      .flatMap(mapPathTo(['**/*.scss'], ({ dir: requirePath, name }) => [
+        join(requirePath, `${name}.scss`),
+        join(requirePath, `${name}.scss.map`) // with source map
```

Comment:
Service teams with source maps enabled will have been inspecting our Autoprefixer-transformed sources, not the original prefix-free `*.scss` source files